README.md

```diff
@@ -290,17 +290,17 @@ The sources can be cloned from GitHub
     - [pymongo][pymongo], optional, (used for mongodb storage plugin)
 
 #### Build dependencies
-  - [cmake][9], required for building
-  - hdf5 development libraries, optional (needed by HDF5 storage plugin)
-  - librdf development libraries, optional (needed by librdf storage plugin)
-  - Python 3 development libraries, optional (needed by Python bindings)
-  - NumPy development libraries, optional (needed by Python bindings)
-  - [SWIG v3][10], optional (needed by building Python bindings)
-  - [Doxygen][11], optional, used for documentation generation
-  - [valgrind][12], optional, used for memory checking (Linux only)
-  - [cppcheck][13], optional, used for static code analysis
-  - [mongomock][mongomock], optional, used for testing mongodb storage plugin
-
+  - [cmake][9], required for building - note that cmake isntalled from pypi does not always work.
+  - hdf5 development libraries, needed by HDF5 storage plugin.
+  - Python 3 development libraries, needed by Python bindings.
```

Comment:
Add link or exact name to be used for apt-get install command?

Reply:
No, this is dependent on the linux distribution. I think we can assume that the person installing dlite for building person knows this for the given system or is able to google it.